collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/CatchFormDesignExceptionsTest.kt

```diff
@@ -23,9 +23,14 @@ class CatchFormDesignExceptionsTest {
         rule.startAtMainMenu()
             .copyForm("form_design_error.xml")
             .clickFillBlankForm()
-            .clickOnForm("form_design_error")
-            .openSelectMinimalDialog()
-            .clickOnText("A")
+            .clickOnForm("Relevance and calculate loop")
+            .answerQuestion(1, "B")
+            .scrollToAndAssertText("third")
```

Comment:
This feels awkward but I couldn't find a better way to make sure that the third question was displayed even on a short screen. Scrolling to the question text is insufficient because the edittext needs to be on-screen.